gcc/rust/ast/passes/rust-feature-gate.h

```diff
@@ -0,0 +1,223 @@
+// Copyright (C) 2020-2022 Free Software Foundation, Inc.
```

Comment:
I might be missing something but why are we creating a pass directory within the AST? I feel like this could  be within the lints pass's directory